### PR TITLE
[ENG-3682] Improve README.md file

### PR DESCRIPTION
**Pull request by An4ik**
AFTER README.md file:

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,10 @@ flake8-f-string is a flake8 plugin to warn passing callable objects as arguments
 
 - Python 3
 - flake8 >= 3.5
+
+Error codes
+-----------
+
+This plugin is using the following error codes:
+
+- ERG100 - passing callable objects as arguments for formatted string
```
Comment (st4lk) [inline README.md]: Propose to not expose company name into error codes.

We are adding "f-string" plugin. So maybe `FS101` would be better?

File: README.md [hunk 2]
```diff
@@ -11,3 +11,10 @@ flake8-f-string is a flake8 plugin to warn passing callable objects as arguments
 
 - Python 3
 - flake8 >= 3.5
+
+Error codes
+-----------
+
+This plugin is using the following error codes:
```
Comment (Abildin) [inline README.md]:
```suggestion
This plugin uses the following error codes:
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Flake8-f-string
 
-flake8-f-string is a flake8 plugin to warn passing callable objects as arguments for formatted string.
+flake8-f-string is a flake8 plugin to warn formatting string `f''`.
```
Comment (Abildin) [inline README.md]:
```suggestion
**flake8-f-string** check and warn about [f-string](https://www.python.org/dev/peps/pep-0498/) issues.
```

**Review by Abildin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (An4ik) [reply README.md]: changed

File: README.md [hunk 4]
```diff
@@ -11,3 +11,10 @@ flake8-f-string is a flake8 plugin to warn passing callable objects as arguments
 
 - Python 3
 - flake8 >= 3.5
+
+Error codes
+-----------
+
+This plugin uses the following error codes:
+
+- FS100 - passing callable objects as arguments for formatted string
```
Comment (st4lk) [inline README.md]: All the error codes start with `101`, not with `100`.

Let's not introduce new rules - keep existing.

Check this (or any plugin):
https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

**Review by st4lk — APPROVED** [review]

**Pull request merged**